Mathlib/Order/LocallyFinite.lean

```diff
@@ -132,7 +132,7 @@ class LocallyFiniteOrder (α : Type _) [Preorder α] where
   finset_mem_Ioo : ∀ a b x : α, x ∈ finsetIoo a b ↔ a < x ∧ x < b
 #align locally_finite_order LocallyFiniteOrder
 
-/-- A locally finite order top is an order where all intervals bounded above are finite. This is
+/-- A locally finite order top is an order where all intervals bounded below are finite. This is
```

Comment:
Should we also have a better informal name for this class? A *locally finite order top* sounds like it is a certain kind of *top* but the docstring claims it is a certain kind of order. Maybe the docstring should simply reflect better that *l.f.o.t.* is a mixin...

Reply:
Rewritten, could you take another look?